### PR TITLE
Chart: changelog for 1.1.0, add UPDATING to docs

### PR DESCRIPTION
**Pull request by jedcunningham**
This PR creates a CHANGELOG for the helm chart, and includes both the CHANGELOG and UPDATING in the docs website so they can be easily consumed.

Below are a few screenshots to give a preview:

nav
updating
changelog2

### CODE REVIEW

File: chart/CHANGELOG.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
+Airflow Helm Chart 1.1.0, 2021-07-25
+------------------------------------
+
+New Features
+""""""""""""
+- Chart: Allow using krb5.conf with ``CeleryExecutor`` (#16822)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- Chart: Allow using ``krb5.conf`` with ``CeleryExecutor`` (#16822)
```

File: chart/CHANGELOG.txt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- Chart: refactor webserver and flower networkpolicy (#16619)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- Chart: Refactor webserver and flower networkpolicy (#16619)
```

File: chart/CHANGELOG.txt [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- Chart: Apply worker's node assigning settings to Pod Template File (#16663)
+- Chart: Support for overriding webserver and flower service ports (#16572)
+- Chart: Support ``extraContainers`` and ``extraVolumes`` in flower (#16515)
+- Chart: Allow configuration of pod resources in helm chart (#16425)
+- Chart: Support job level annotations; fix jobs scheduling config (#16331)
+- feat: Helm chart adding minReplicaCount to the Keda worker-kedaautoscaler.yaml (#16262)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- feat: Helm chart adding ``minReplicaCount`` to the Keda ``worker-kedaautoscaler.yaml`` (#16262)
```

File: chart/CHANGELOG.txt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- Chart: Adds support for custom command and args (#16153)
+- Chart: Add extra ini config to ``pgbouncer`` (#16120)
+- Chart: Add ``extraInitContainers`` to scheduler/webserver/workers (#16098)
+- Configurable resources for git-sync sidecar (#16080)
+- Chart: Template ``airflowLocalSettings`` and ``webserver.webserverConfig`` (#16074)
+- Support ``strategy``/``updateStrategy`` on scheduler (#16069)
+- Chart: Add both airflow and extra annotations to jobs (#16058)
+- `loadBalancerIP` and `annotations` for both Flower and Webserver (#15972)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- ``loadBalancerIP`` and ``annotations`` for both Flower and Webserver (#15972)
```

File: chart/CHANGELOG.txt [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Improvements
+""""""""""""
+- Chart: Update postgres subchart to 10.5.3 (#17041)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- Chart: Update Postgres subchart to 10.5.3 (#17041)
```

File: chart/CHANGELOG.txt [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+- Chart: Update the default Airflow version to ``2.1.2`` (#17013)
+- Update default image as ``2.1.1`` for Helm Chart (#16785)
+- Chart: warn when using default logging with KubernetesExecutor (#16784)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- Chart: warn when using default logging with ``KubernetesExecutor`` (#16784)
```

File: chart/CHANGELOG.txt [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+Doc only changes
+""""""""""""""""
+- Chart Docs: Add single-line description for ``multiNamespaceMode`` (#17147)
+- Chart: Update description for Helm chart to include 'official' (#17040)
+- Chart: Better comment and example for `podTemplate` (#16859)
+- Chart: Add more clear docs for setting `pod_template_file.yaml` (#16632)
+- Fix description on scheduler.livenessprobe.periodSeconds (#16486)
+- Chart docs: Fix ``extrasecrets`` example (#16305)
+- Small improvements for README.md files (#16244)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- Small improvements for ``README.md`` files (#16244)
```

File: chart/CHANGELOG.txt [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+- Handle special characters in password sfor Helm Chart (#16004)
+- Fix flower serviceAccount created without flower enable (#16011)
+- Chart: ``gitsync`` Clean Up for ``KubernetesExecutor``  (#15925)
+- Mount DAGs read only when using ``gitsync`` (#15953)
+
+Doc only changes
+""""""""""""""""
+- Chart Docs: Add single-line description for ``multiNamespaceMode`` (#17147)
+- Chart: Update description for Helm chart to include 'official' (#17040)
+- Chart: Better comment and example for `podTemplate` (#16859)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- Chart: Better comment and example for ``podTemplate`` (#16859)
```

File: chart/CHANGELOG.txt [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+- Chart: Add more clear docs for setting `pod_template_file.yaml` (#16632)
+- Fix description on scheduler.livenessprobe.periodSeconds (#16486)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
- Fix description on ``scheduler.livenessprobe.periodSeconds`` (#16486)
```

File: chart/CHANGELOG.txt [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+- Small improvements for README.md files (#16244)
+
+Misc
+""""
+- Removes pylint from our toolchain (#16682)
+- Update link to match what is in pre-commit (#16408)
+- Chart: Update the ``appVersion`` to 2.1.0 in ``Chart.yaml`` (#16337)
+- Rename the main branch of the Airflow repo to be `main` (#16149)
+- Update Chart version to ``1.1.0-rc1`` (#16124)
+- Updates branches and branch documentation after 2.1.0rc1 (#15528)
```
Comment (kaxil) [inline chart/CHANGELOG.txt]:
```suggestion
```

**Review by kaxil — APPROVED** [review]

File: chart/CHANGELOG.txt (hunk 1 above)
Comment (jedcunningham) [reply chart/CHANGELOG.txt]: Thanks @kaxil. For some reason I figured the commit messages should be left as-is. I'll keep this in mind for next time.

**Pull request merged**